zerver/openapi/zulip.yaml

```diff
@@ -16976,6 +16988,38 @@ components:
                   The unique message ID. Messages should always be
                   displayed sorted by ID.
     BasicStream:
+      allOf:
+        - $ref: "#/components/schemas/BasicStreamBase"
+        - additionalProperties: false
+          properties:
+            stream_id: {}
+            name: {}
+            description: {}
+            date_created: {}
+            invite_only: {}
+            rendered_description: {}
+            is_web_public: {}
+            stream_post_policy: {}
+            message_retention_days:
+              nullable: true
+            history_public_to_subscribers: {}
+            first_message_id:
+              nullable: true
+            is_announcement_only: {}
+            can_remove_subscribers_group: {}
+            stream_weekly_traffic:
+              type: integer
+              nullable: true
+              description: |
+                The average number of messages sent to the stream in recent weeks,
+                rounded to the nearest integer.
+
+                If `null`, the stream was recently created and there is
+                insufficient data to estimate the average traffic or it
+                is not required to send the traffic data to clients.
+
+                **Changes**: New in Zulip 8.0 (feature level 199).
+    DefaultStream:
```

Comment:
This is also a temporary workaround for not including `stream_weekly_traffic` field in default stream and default stream groups objects.